src/shared/pages/team/deactivateTeam/deactivateTeam.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
I wonder if we should be blocking this page from being loaded and redirect if the user doesn't have the auth to visit? As they could manipulate the url to visit this page.

Reply:
I have addressed this.

Reply:
Has it? I was thinking you bring in the hasRole, use it in the core `useEffect()` method to check for the permission and redirect if it's not found? The same would then be done in the activateTeam page to do the same?